src/main/java/Duke.java

```diff
@@ -1,10 +1,294 @@
+import task.*;
```

Comment:
Should the * import can be replaced with the individual required imports? I think it was in the coding standard guidelines.

Reply:
Got it! Thank you for pointing it out :)